teku/src/main/java/tech/pegasys/teku/cli/subcommand/internal/validator/tools/KeyGenerator.java

```diff
@@ -56,6 +57,10 @@ public KeyGenerator(
       final CommandSpec commandSpec,
       final Function<String, String> envSupplier,
       final Consumer<String> commandOutput) {
+    if (validatorCount < 0) {
+      throw new InvalidConfigurationException(
+          String.format("Invalid validatorCount: %d", validatorCount));
+    }
```

Comment:
I'm unsure if this check belongs here. It's the `--number-of-validators` option in `KeyGenerationOptions`.

Reply:
That value is only used by some debug subcommands which are useful for testing but not recommended for users, so doesn't have the usual config object pattern.  So I'd say this is probably the best place to put the check.